### PR TITLE
built the product component of ecommerce app

### PR DESCRIPTION
**Pull request by devAmlan**
Hey guys, I built the product component of my ecommerce app. Please review my PR.

### CODE REVIEW

**Review by zoef-yt — APPROVED** [review]
The code looks good, add a better description.

File: frontend/src/api/db/Productdb.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import kb3 from "../../assets/img/kb3.jpg"
+import kb4 from "../../assets/img/kb4.jpg"
+import k1 from "../../assets/img/k1.jpg"
+import k2 from "../../assets/img/k2.jpg"
+import k3 from "../../assets/img/k3.jpg"
+import k4 from "../../assets/img/k4.jpg"
+import deskpad from "../../assets/img/deskpad.png"
+const Productdb = [
+  {
+    id:0,
```
Comment (sudipkundu999) [inline frontend/src/api/db/Productdb.js:12]: To work with Mockbee it's necessary for items to have a `_id`. So add that while putting this in db.

File: frontend/src/pages/productpage/ProductPage.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import { Productdb } from '../../api/db/Productdb'
+import Product from '../../components/product/Product'
+function ProductPage() {
+  const [products] = useState(Productdb)
```
Comment (sudipkundu999) [inline frontend/src/pages/productpage/ProductPage.js]: Can you please explain what are you doing here?
Why put `Productdb` in a state and that too when you're not using the state's setter function?

You can directly use Productdb and map over it (as it is an array of objects)

File: frontend/src/api/db/Productdb.js (hunk 1 above)
Comment (devAmlan) [reply frontend/src/api/db/Productdb.js:12]: thanks sudip , i will change it.

File: frontend/src/pages/productpage/ProductPage.js (hunk 2 above)
Comment (devAmlan) [reply frontend/src/pages/productpage/ProductPage.js]: ok I will do that , thanks

**Pull request merged**